debezium-ddl-parser/src/test/resources/mysql/examples/ddl_alter.sql

```diff
@@ -1,54 +1,49 @@
 #begin
 -- Alter Table
-alter table ship_class add column ship_spec varchar(150) first, add somecol int after start_build, algorithm=instant;
+alter table ship_class add column ship_spec varchar(150) first, add somecol int after start_build;
```

Comment:
The `algorithm=instant` is valid for MySQL.

I understand the goal is to synchronize with upstream, but we are doing a disservice to ourselves if we take the upstream as-is without comparing the differences and those that are valid being retained downstream and sent upstream as a part of this effort. That's the only way we can avoid introducing any potential grammar regressions, no?

I haven't checked all the example SQL files, but we likely should.

Reply:
The idea was to synchronize with upstream and add the remaining downstream-specific examples/grammars to a separate file. So, I have added them to a new file, `debezium_tests.sql`.

However, this would cause grammar duplication with subtle differences in the two files.

Reply:
So ultimately I'll defer to @jpechane if he disagrees, but this `debezium_tests.sql` file would ideally only be the bits that are different between upstream/downstream that apply to things like Percona or unusual recordings of DDL that we say cannot confirm via the documented grammar. For example, think of the work we did for the Oracle data pump and data loader rules we added for it.

Ideally, if the documented grammar supports something, such as the `algorithm=instant` bit, that I would think belongs in the appropriate upstream SQL test files and contributed upstream, no?

Reply:
Yes, @Naros, you are correct. After discussing this with Jiri yesterday, we clarified the final changes. I will contribute the differences to the upstream and add the specific downstream extensions to `debezium_tests.sql`. Additionally, I will separate the Oracle formatting into a separate PR and work on it after we have resolved the MariaDB and MySQL grammars.